mdapi/database/main.py

```diff
@@ -247,30 +253,35 @@ def index_repositories():
             "{dlserver}/pub/epel/{versname}/Everything/x86_64/repodata/",
             "{dlserver}/pub/epel/testing/{versname}/Everything/x86_64/repodata",
         ],
+        "Fedora EPEL Next": [
+            "{dlserver}/pub/epel/next/{versname}/Everything/x86_64/repodata/",
+            "{dlserver}/pub/epel/next/testing/{versname}/Everything/x86_64/repodata",
+        ],
     }
 
-    urls["Fedora"] = urls["Fedora Linux"]
-
     repodict = {
         "fedora": ["{rlid}", "{rlid}-updates", "{rlid}-updates-testing"],
-        "epel": ["{rlid}", "{rlid}-testing"]
+        "epel": ["{rlid}", "{rlid}-testing"],
+        "epel-next": ["{rlid}", "{rlid}-testing"],
     }
 
     # Obtain the stable repos
-    stable_releases = list_branches(status="Active")
-    for rels in stable_releases:
-        if rels["status"] != "Active":
+    for rels in list_branches(status="current"):
+        if not re.search(r"f[0-9]+", rels) or not re.search(r"epel[0-9]+", rels) or not re.search(r"epel[0-9]+-next", rels):  # noqa : E501
             continue
-        versdata = rels["version"]
-        for jndx, urli in enumerate(urls[rels["name"]]):
-            if rels["name"] in ("Fedora Linux", "Fedora"):
-                name = repodict["fedora"][jndx].format(rlid = rels["koji_name"])
-            elif rels["name"] == "Fedora EPEL":
-                name = repodict["epel"][jndx].format(rlid = rels["koji_name"])
-            else:
-                # TODO: Do we need to have the the above elif with this else case?
-                name = repodict["epel"][jndx].format(rlid = rels["koji_name"])
-            rurl = urli.format(dlserver = standard.DL_SERVER, versname = versdata)
+        versdata = re.search(r"\d+", rels).group()
```

Comment:
This regex will also need to account for future epel10 minor versions.

```suggestion
        versdata = re.search(r"\d+(?:\.\d+)?", rels).group()
```